setup.cfg

```diff
@@ -35,6 +35,22 @@ packages = find:
 include_package_data = true
 zip_safe = false
 python_requires = >= 3.6
+install_requires =
+    gevent >=20.12.1
+    flask >=2.0.0
+    Werkzeug >=2.0.0
+    requests >=2.23.0
+    msgpack >=0.6.2
+    pyzmq >=22.2.1
+    geventhttpclient >=1.5.1
+    ConfigArgParse >=1.0
+    psutil >=5.6.7
+    Flask-BasicAuth >=0.2.0
+    Flask-Cors >=3.0.10
+    roundrobin >=0.0.2
+    typing-extensions >=3.7.4.3
+    Jinja2 <3.1.0
+    pywin32;platform_system=='Windows'
```

Comment:
i was unable to get it working with `pywin32` being an  `extras_require` as in the `setup.py` case, however this *should* be the same(?). pywin32 only needed when running on windows.
https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#platform-specific-dependencies